sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryAvroUtils.java

```diff
@@ -47,6 +47,19 @@
  */
 class BigQueryAvroUtils {
 
+  public static final ImmutableMap<String, Type> BIG_QUERY_TO_AVRO_TYPES =
```

Comment:
Did you mean to edit this file and the other BQ/BigTable files in this PR?
What problem were you trying to solve?